builds/misc/images.yaml

```diff
@@ -31,9 +31,9 @@ jobs:
       - bash: 'cargo install --git https://github.com/arsing/cross.git --branch set-path'
         displayName: 'Install cross (fork with docker fix)'
 
-      - script: scripts/linux/buildBranch.sh -c Release --no-rocksdb-bin --os Unix --dotnet_runtime netcoreapp3.0
+      - script: scripts/linux/buildBranch.sh -c $(Build.Configuration) --no-rocksdb-bin --os Unix --dotnet_runtime netcoreapp3.0
         name: build
-        displayName: Build (release)
+        displayName: Build ($(Build.Configuration))
 
       - script: scripts/linux/buildDiagnostics.sh
```

Comment:
The diagnostics tool is always built with release config (in `buildDiagnostics.sh` and `build-diagnostics.ps1`). Is that OK? Or do we also need to make that build configurable?

Reply:
Update these script to take build configuration from parameter; you can verify from this build, https://msazure.visualstudio.com/One/_build/results?buildId=21356904.